# Housing Market

### Introduction:

This time we will create our own dataset with fictional numbers to describe a house market. As we are going to create random data don't try to reason of the numbers.

### Step 1. Import the necessary libraries

In [2]:
import numpy as np 
import pandas as pd

### Step 2. Create 3 differents Series, each of length 100, as follows: 
1. The first a random number from 1 to 4 
2. The second a random number from 1 to 3
3. The third a random number from 10,000 to 30,000

In [4]:
ser1  = pd.Series(np.random.randint(1,4,100))
ser2  = pd.Series(np.random.randint(1,3,100))
ser3  = pd.Series(np.random.randint(10000,30000,100))

### Step 3. Let's create a DataFrame by joinning the Series by column

In [8]:
df = pd.concat([ser1,ser2,ser3],axis="columns")
df

0  1      2
0   3  2  23946
1   2  2  27493
2   1  1  28238
3   3  1  27467
4   1  1  25432
.. .. ..    ...
95  3  1  14444
96  3  2  27081
97  2  1  19663
98  1  1  18228
99  2  2  20301

[100 rows x 3 columns]

### Step 4. Change the name of the columns to bedrs, bathrs, price_sqr_meter

In [9]:
df.columns = ['bedrs', 'bathrs', 'price_sqr_meter']
df

bedrs  bathrs  price_sqr_meter
0       3       2            23946
1       2       2            27493
2       1       1            28238
3       3       1            27467
4       1       1            25432
..    ...     ...              ...
95      3       1            14444
96      3       2            27081
97      2       1            19663
98      1       1            18228
99      2       2            20301

[100 rows x 3 columns]

### Step 5. Create a one column DataFrame with the values of the 3 Series and assign it to 'bigcolumn'

In [11]:
bigcolumn = pd.concat([ser1,ser2,ser3]).reset_index(drop=True)
bigcolumn

0          3
1          2
2          1
3          3
4          1
       ...  
295    14444
296    27081
297    19663
298    18228
299    20301
Length: 300, dtype: int32

### Step 6. Oops, it seems it is going only until index 99. Is it true?

### Step 7. Reindex the DataFrame so it goes from 0 to 299